microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,591 @@
+---
+title: Get started with WebView2 in HoloLens 2 Unity apps
+description: Get started with WebView2 in HoloLens 2 Unity apps.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/01/2023
+---
+# Get started with WebView2 in HoloLens 2 Unity apps (Preview)
+
```

Comment:
@michaelfarnsworth - pls review proposed long name & short name of plugin.  I believe there's only 1 plugin referred to by these names.  After you confirm a long name and short name to use, it should be clear what changes to make.

Proposed consistent long name:
the Microsoft Mixed Reality WebView plugin

Proposed consistent short name:
the WebView plugin for Unity
_____________
Names found in draft:

Long name:

the **Microsoft Mixed Reality WebView** plugin -- bold ok for introducing.
the "Microsoft Mixed Reality WebView" plugin -- remove quotes or (less likely) add bold.
the Microsoft Mixed Reality WebView plugin
the Microsoft Mixed Reality WebView plugin
the Microsoft Mixed Reality WebView plugin
the Microsoft Mixed Reality WebView plugin
the Microsoft Mixed Reality WebView plugin

Short name:

the WebView plugin for Unity  <-- best, clearest
the WebView Unity plugin <-- change to above
Unity WebView plugin <-- change to above

Super-short name, add "for Unity" suffix as context specifier:

the WebView plugin
the WebView plugin
the WebView plugin
the WebView plugin
the WebView plugin

Reply:
The proposals look good to me.

Reply:
implemented in .md globally.
you can Resolve this thread